lib/driver.js

```diff
@@ -15,6 +15,7 @@ import { detectUdid, getAndCheckXcodeVersion, getAndCheckIosSdkVersion,
          adjustWDAAttachmentsPermissions, checkAppPresent, getDriverInfo,
          clearSystemFiles, translateDeviceName, normalizeCommandTimeouts,
          DEFAULT_TIMEOUT_KEY } from './utils';
+import { killAppUsingAppName } from './wda/utils';
```

Comment:
If this is being used outside of the `wda` folder it should be moved into the root `utils.js`.